recipes/p7zip/all/test_package/conanfile.py

```diff
@@ -0,0 +1,11 @@
+from conan import ConanFile
+from conan.tools.build import can_run
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "VirtualRunEnv"
+
+    def test(self):
```

Comment:
I believe these are needed for 2.0 client

```suggestion
    generators = "VirtualRunEnv"
    test_type = "explicit"

    def build_requirements(self):
        self.tool_requires(self.tested_reference_str)

    def test(self):
```

Reply:
https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/prebuilt_tool_package/all/test_package/conanfile.py